Update to rails 6.1

beccapearce opened the pull request:
Steps taken:
- Run rails app:update
- Fix linting errors raise when running rubocop

Trello card: https://trello.com/c/nWRFaLO3/2562-manually-bump-apps-to-rails-61

brucebolt commented on config/application.rb:
```diff
@@ -18,9 +24,12 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
```
```suggestion
    config.load_defaults 6.1
```

brucebolt commented on config/environments/development.rb:
```diff
@@ -64,4 +75,6 @@
   config.hosts += %w[
     local-links-manager.dev.gov.uk
   ]
+  # Uncomment if you wish to allow Action Cable access from any origin.
```
It's a real nitpick, but could you add an empty line above this comment, so it is easier to see this is a separate configuration to the one above.

brucebolt reviewed (commented):
Looks good to me.  I've just left a couple of really tiny suggestions.

beccapearce committed:
run rails app:update

update the app with the new rails 6.1 config

beccapearce committed:
Uncomment relevant defaults from rails 6.1

Uncomment the new defaults from rails 6.1 that apply to this 
application.

The ones still commented result in no method error.

beccapearce committed:
Use the rails 6.1 defaults

Switch to using the 6.1 defaults and delete the 6.1 framework defaults 
file.
